pkg/daemon/ceph/osd/volume_test.go

```diff
@@ -260,6 +268,55 @@ var cephVolumeRAWTestResult = `{
 }
 `
 
+func TestInitializeBlockPVCResult(t *testing.T) {
+	executor := &exectest.MockExecutor{}
+	executor.MockExecuteCommandWithCombinedOutput = func(command string, args ...string) (string, error) {
+		logger.Infof("%s %v", command, args)
+		if args[1] == "ceph-volume" && args[2] == "raw" && args[3] == "prepare" && args[4] == "--bluestore" && args[7] == "--crush-device-class" {
```

Comment:
We should also check for the `--block.db` arguments when `metadata` device is enabled.  [Reference](https://github.com/rook/rook/blob/1ea73ce1233fcaa4a1b5518967eef2dcbd0cade6/pkg/daemon/ceph/osd/volume.go#L261)